script.js

```diff
@@ -1,35 +1,259 @@
+  languages: ['Html', 'Ruby on rails', 'JavaScrip', 'CSS'],
+  seeLink: 'https://damiandib9.github.io/averger/',
+  sourceLink: 'https://github.com/DamianDib9/averger',
+},
+];
+
+const button2 = document.querySelectorAll('.STP-button');
+const popupContainerDesk = document.querySelector('.popupContainer2');
+
+button2.forEach((i, indice2) => {
+  i.addEventListener('click', () => {
+    const damian2 = `<div class="superPopUpCont">
+    <div class="popupTextBox">
+          <div class="popboxTitle">
+              <h3 id="crossTypo2" class="popProjectTitle"> <img class="xcrosstypo" src="img/menudesp/crossMENU.svg" alt="close"> </h3>
+              <h3 class="popProjectTitle"> ${popupProjectsDesk[indice2].projectName} </h3>
+          </div>
+          <div class="popCodingLangButtons">
+              <ul class="lists">
+                  <li class="popupLang">${popupProjectsDesk[indice2].languages[0]}</li>
+                  <li class="popupLang">${popupProjectsDesk[indice2].languages[1]}</li>
+                  <li class="popupLang">${popupProjectsDesk[indice2].languages[2]}</li>
+              </ul>
+          </div>
+      </div>
+  
+      <div class="popPicture">
+          <img class="popPicturesPreview" src="${popupProjectsDesk[indice2].picture}" alt="Project Number 1">
+      </div>
+      <div class="multiPictures">
+      <img class="multiPictures25" src="${popupProjectsDesk[indice2].picture}" alt="Project Number 1"> <img class="multiPictures25" src="${popupProjectsDesk[indice2].picture}" alt="Project Number 1"> <img class="multiPictures25" src="${popupProjectsDesk[indice2].picture}" alt="Project Number 1"> <img class="multiPictures25" src="${popupProjectsDesk[indice2].picture}" alt="Project Number 1">
+  </div>
+  
+      <div id="popupPdiv">
+          <p class="popupP">
+          ${popupProjectsDesk[indice2].description}
+          </p>
+  
+          <div class="inlineBlockButton">
+          <a href="${popupProjectsDesk[indice2].seeLink}">
+              <button type="button" id="popUnionButton" class="popupButtons">
+                  <h4 class="popButText">See live<img class="imgBTN" src="img/popup/unionPopup.svg" alt=""></h4>
+                  
+              </button>
+              </a>
+
+              <a href="${popupProjectsDesk[indice2].sourceLink}">
+              <button type="button" id="popGitHubButton" class="popupButtons">
+                  <h4 class="popButText"> See source  <img class="imgBTN" src="img/popup/GithubPopup.svg" alt=""></h4>
+              </button>
+              </a>
```

Comment:
- [ ] Please use the same background color on the active, hover or ... as provided and this will help you to fulfill the design requirements.


## View:
![image](https://user-images.githubusercontent.com/50990925/226151898-23138038-887f-4719-b7d7-96bcc39ebb22.png)


![image](https://user-images.githubusercontent.com/50990925/226151876-7b2644ea-4566-43b9-a3e3-c601a2f9e801.png)

## Figma

Reply:
Done!